src/style.css

```diff
@@ -2,0 +1,96 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir.ttc) format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Book', sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy', sans-serif;
+  src: url(/src/fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 800;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
The `font-family` property in the `body` selector should match the custom font name defined in the `@font-face` rule, without including fallback fonts like 'sans-serif'.